src/components/Simulator/index.tsx

```diff
@@ -0,0 +1,205 @@
+
+  const nextPathSummary = useMemo<SimulatorPathSummary | null>(() => {
+    if (!simulator) return null
+    const { nextPathId } = simulator
+    if (!nextPathId) return null
+    const { title, Icon } = data[nextPathId]
+    return {
+      primaryText: "Start next lesson",
+      secondaryText: title,
+      Icon,
+    }
+  }, [pathId])
+
+  const logFinalCta = (): void => {
+    trackCustomEvent({
+      eventCategory: "simulator",
+      eventAction: `${pathId}_click`,
+      eventName: `find-wallet`,
+    })
+  }
+
+  return (
+    <Grid
+      id={SIMULATOR_ID}
+      bg="cardGradient"
+      placeItems="center"
+      p={{ base: 4, md: 16 }}
+      w="full"
+      scrollMarginTop={{ base: "5rem" }}
+      scrollBehavior="smooth"
+    >
+      <Flex
+        direction={{ base: "column", md: "row" }}
+        bg="background.base"
+        px={{ base: 4, md: 16 }}
+        py={{ base: 8, md: 16 }}
+        alignItems="center"
+        textAlign={{ base: "center", md: "start" }}
+        gap={{ base: 16, md: 8, lg: 16 }}
+        maxW="1000px"
+        w="full"
+      >
+        {/* TEXT CONTENT */}
+        <Flex direction="column" px={4}>
+          {children}
+        </Flex>
+        {/* Button stack for path options */}
+        <Flex
+          direction="column"
+          gap={8}
+          w={{ base: "min(100%, 320px)", md: "300px" }}
+          minW={{ md: "300px" }}
+        >
+          {Object.keys(data).map((id) => {
+            const sim = data[id]
+            const pathSummary = {
+              primaryText: sim.title,
+              secondaryText: "How to?",
+              Icon: sim.Icon,
+            }
+            return (
+              <PathButton
+                key={id}
+                pathSummary={pathSummary}
+                handleClick={() => {
+                  trackCustomEvent({
+                    eventCategory: "simulator",
+                    eventAction: `main-buttons_click`,
+                    eventName: id,
+                  })
+                  openPath(id as PathId)
+                }}
+              />
+            )
+          })}
+        </Flex>
+      </Flex>
+      <SimulatorModal isOpen={isOpen} onClose={onClose}>
+        {isOpen && Screen && (
+          <Template>
+            <Explanation
+              nav={nav!}
+              explanation={explanation!}
+              nextPathSummary={nextPathSummary}
+              nextPathId={nextPathId ?? null}
+              finalCtaLink={finalCtaLink!}
+              openPath={(id: PathId) => {
+                trackCustomEvent({
+                  eventCategory: "simulator",
+                  eventAction: `${pathId}_click`,
+                  eventName: `next-lesson-${id}`,
+                })
+                openPath(id)
+              }}
+              logFinalCta={logFinalCta}
+            />
+            <Phone>
+              <Screen nav={nav!} ctaLabel={ctaLabel!} />
+            </Phone>
```

Comment:
A question for future iterations about re-usability: how could we refactor this part since another simulator might not want to use a `Phone` layout?

It could be part of the given `Screen`.

Reply:
Yeah, or perhaps we create a prop for this `Simulator` component called `Display`, and pass `Phone` through from the parent, which could be swapped out for say `BrowserWindow`...
Adding it to the Screen would be somewhat redundant, but would offer the flexibility of switching displays potentially within one sim.